control/control.go

```diff
@@ -101,8 +85,10 @@ func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*
 
 	var expi exporter.ExporterInstance
 	var err error
+	// FIXME mw0
+	mw0 := c.opt.MetaWorkers[0]
```

Comment:
This is actually tricky, as the exporter should come from the worker that has the returned reference. We may need to delay this so that the solver loads this.